openrndr-application/src/jvmMain/kotlin/org/openrndr/Application.kt

```diff
@@ -87,20 +79,39 @@ actual abstract class Application {
                 logger.debug { "NullGL not found" }
             }
 
-            return if (!configuration.headless)
-                Application::class.java.classLoader.loadClass("org.openrndr.internal.gl3.ApplicationGLFWGL3")
-            else
-                Application::class.java.classLoader.loadClass("org.openrndr.internal.gl3.ApplicationEGLGL3")
+            val applicationProperty: String? = System.getProperty("org.openrndr.application")
+
+            @Suppress("KotlinConstantConditions")
+            return when (applicationProperty) {
+                null, "", "ApplicationGLFW" -> Application::class.java.classLoader.loadClass("org.openrndr.internal.gl3.ApplicationGLFWGL3")
+                "ApplicationEGL" -> Application::class.java.classLoader.loadClass("org.openrndr.internal.gl3.ApplicationEGLGL3")
+                else -> throw IllegalArgumentException("Unknown value '${applicationProperty}' provided for org.openrndr.application")
```

Comment:
Are these names intuitive for the user? Should we instead simply use "GLFW" and "EGL"?